README.md

```diff
@@ -28,6 +28,12 @@ Imagine having a central hub where you can create and send notifications seamles
 2. **Create Notifications:** Easily create and customize notifications for different channels. Use your own API or our API to update the notification database.
 3. **Sit Back and Relax:** Let our app handle the delivery and status tracking.
 
+## Documentation
+
+- [Development Setup](docs/development-setup.md)
+- [Production Setup](docs/production-setup.md)
```

Comment:
Don't add links for document which are not ready. When they are ready we will add it.

Reply:
see it is removed.